README.md

```diff
@@ -133,6 +133,17 @@ If you want to build only one of the services, you can launch:
 
 Commits follow the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/).
 
+### Pre-commit
+The precommit hook use gradle to automatically apply detekt rules.
+It will not be able to work if gradle is not in your $path variable
```

Comment:
since the script is calling `./gradlew`, I think it is not needed to install Gradle beforehand (it will be done when first calling `gradlew`: https://docs.gradle.org/current/userguide/gradle_wrapper.html).